web/src/components/avatar/group-item/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { JssStyle } from 'jss'
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+import { parseToken } from '../../../utils'
+
+import { GroupItemProps } from '.'
+
+export default createUseStyles(
+  (theme: typeof tokens): Record<string, JssStyle> => ({
+    name: {
+      color: ({ isInactive }: GroupItemProps) =>
+        isInactive
+          ? theme.colorTextAvatarItemPrimaryInactive
+          : theme.colorTextAvatarItemPrimary,
+    },
+    secondary: {
+      display: 'flex',
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      marginLeft: ({ size }: GroupItemProps) =>
+        size === 'small' ? theme.spaceAvatarItem : 0,
+    },
+    description: {
+      color: theme.colorTextAvatarItemSecondary,
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      textAlign: 'left',
```

Comment:
again duplicated. please do the same with name!

Reply:
Thanks, just pushed updates